docs/source/background/scf_notes.md

```diff
@@ -0,0 +1,86 @@
+<!--
+  ~ Copyright 2025 NWChemEx-Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+
+# SCF Procedure
+
+## SCF Driver
+
+1. Obtain the AO parameters as well as the chemical system (inputs)
+2. Assign the AO parameters as atomic orbitals
+3. Create the Hamitonian from the Chemical System (Submod "Hamiltonian" used, currently only the BO Approximation from NUX)
+    1. For the BO Approximation Hamiltonian:
+        1. From the Chemical System, the Electrons and Nuclei are used to form the Hamiltonian
+        2. Each electron gets a 1 electron kinetic energy term
+        3. Each nuclei gets an Electron-Nuclei potential energy term
+        4. If more than 1 electron, an Electron-Electron potential energy repulson term is added
+        5. If more than 1 nuclei, a Nuclei-Nuclei potential energy repulsion term is added
+        6. In order, the Electron kinetic energy, Electron-nucleus attraction energy, and Electron-electron repulsion energy are
+           formed to create the Electronic Hamiltonian
+        7. The Nuclear-nuclear repulsion energy is also created
+        8. The BO Approximation returns the constructed Hamiltonian, H.
+4. With the guess submod, (of type InitialGuess), the inputs are the Hamiltonian and the AOs, and the return value is the
+   initial wavefunction
```

Comment:
Here I'm thinking something more like:

```
Given the Hamiltonian for the system and the AO basis set, we are able to compute a trial wavefunction for the SCF procedure. This is done by calling a module of type `InitialGuess`. This is where options such as core guess, superposition of atomic densities, etc. are used with the SCF.
```
We don't want the core guess to be the long-term default, so I'd avoid saying it's the default. Again, the details of what happens in the module should be described in the documentation for the module.